packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -0,0 +1,234 @@
+var fs = require('fs');
+var path = require('path');
+var childProcess = require('child_process');
+var flowBinPath = require('flow-bin');
+const flowTypedPath = path.join(__dirname, 'node_modules', '.bin', 'flow-typed');
```

Comment:
Yes, it would ultimately be used here if an official JS API was exposed, keep me updated on this.